Sources/armory/renderpath/Inc.hx

```diff
@@ -224,8 +223,9 @@ class Inc {
 		#if arm_debug
 		endShadowsLogicProfile();
 		#end
-		#end // rp_shadowmap
+		#end //rp_shadowmap
```

Comment:
Please try to minimize the formatting changes. If there are formatting errors then a separate PR is still welcome.